js-Random-Images.js

```diff
@@ -0,0 +1,12 @@
+const imgButton = document.querySelector('#randomize');
+
+const  myImg= function() {
+
+    const newImg = document.createElement('img');
+
+    newImg.setAttribute('src','https://source.unsplash.com/random');  
```

Comment:
to get a random image with different hights and whidth you take the values from the DOM and assign them to a variable then use those variables in the API URL 

similar to this

```
let width = document.querySelector("input.width");
let height = document.querySelector("input.height");
let url = ("https://source.unsplash.com/" + width + "x" + height);
let image = document.querySelector(".img");
image.setAttribute("src", url);
```